src/library.js

```diff
@@ -300,34 +300,20 @@ LibraryManager.library = {
       return -1;
     }
   },
+
   getpagesize: function() {
     // int getpagesize(void);
-#if MINIMAL_RUNTIME
-#if WASM
-    return {{{ WASM_PAGE_SIZE }}};
-#else
-    return {{{ ASMJS_PAGE_SIZE }}};
-#endif
-#else
-    return PAGE_SIZE;
-#endif
+    return {{{ FS_PAGE_SIZE }}};
```

Comment:
getpagesize is a posix function that is supposed to return virtual memory page size.. so why is returning the VFS one?

Reply:
For example won't brk()/malloc() use this value?

Reply:
This restores the behavior before that PR, so this is just a revert really. (But it also refactors the value into something we can `{{{ }}}` for efficiency.)

This page size is used in the filesystem layer, yeah. I don't actually think it matters much in practice - malloc won't use it, as we use sbrk() there, and it doesn't limit itself to pages. But perhaps user code might, so I went back to the behavior we've had all along.

I don't remember well why this value was picked all the years ago, but I assume it was to make it similar to page sizes in existing OSes?